src/lib.rs

```diff
@@ -211,8 +211,7 @@ impl MqttTrigger {
                             msg.payload().to_vec(),
                             msg.topic().to_owned(),
                         )
-                        .await
-                        .map_err(|err| tracing::error!("{err}"));
+                        .await?;
```

Comment:
@calebschoepp can you advice on what is the best way to still propagate this trace?

Reply:
It's fine for you to still import `tracing` in `Cargo.toml` and keep this line of code. You just shouldn't import `tracing-subscriber`.